flow/workflows/cdc_flow.go

```diff
@@ -91,6 +91,9 @@ func processCDCFlowConfigUpdate(
 		state.SyncFlowOptions.NumberOfSyncs = 0
 	}
 	if flowConfigUpdate.UpdatedEnv != nil {
+		if cfg.Env == nil {
```

Comment:
I'm not sure if we should skip adding the env entirely in this case, need to think it through